skbio/sequence/_grammared_sequence.py

```diff
@@ -289,19 +291,33 @@ def nondegenerate_chars(cls):
             Non-degenerate characters.
 
         """
+        return cls.definite_chars
+
+    @abstractproperty
+    @classproperty
+    @stable(as_of='0.4.0')
```

Comment:
This should be `0.4.2-dev` since the new name will be stable in the next release.